### PR TITLE
React Smooth Scroll -> React Ui

### PR DESCRIPTION
**Pull request by saidMounaim**
# React Smooth Scroll

React Provider Component to add a smooth scroll effect

Link: https://github.com/saidMounaim/React-Use-Smooth-Scroll

#### Checklist:

- [x] I have performed a self-review of submitted resource and its follows the guidelines of the project.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -750,6 +750,7 @@
 | [xstyled](https://xstyled.dev/)| A CSS-in-JS framework built for React with props based utilities like display, fontSize or rotate. |
 | [@use-gesture](https://github.com/pmndrs/use-gesture)| Bread n butter utility for component-tied mouse/touch gestures in React. |
 | [React Rainbow](https://react-rainbow.io/)| React Rainbow is a collection of components that will reliably help you build your application in a snap. |
+| [React Use Smooth Scroll](https://github.com/saidMounaim/React-Use-Smooth-Scroll)| React Provider Component to add a smooth scroll effect 😍 |
```
Comment (Alaev) [inline readme.md:753]: Hey, 
Thanks for the PR, Can you please remove the emoji?

**Review by Alaev — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (saidMounaim) [reply readme.md]: Hello,
Done.

**Comment by Alaev** [comment]
@saidMounaim Can you resolve the conflicts?

**Comment by saidMounaim** [comment]
> @saidMounaim Can you resolve the conflicts?

Done

**Pull request merged**